test/simplifyExpression/basicsSearch/handleDivisionByZero.test.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
```

Comment:
idk how helpful these tests are - in every logic flow (error or not error) we still return the same node unchanged. I think just having the oneStep tests is good. Or you could put those in this file